EpicLoot/LootRoller.cs

```diff
@@ -262,6 +255,7 @@ private static List<GameObject> RollLootTableInternal(LootTable lootTable, int l
                             {
                                 foreach (var itemAmountConfig in disenchantProducts)
                                 {
+                                    EpicLoot.Log($"[RollLootInternal] Step 9 - {ld.Item} - Amount: {itemAmountConfig.Amount}");
```

Comment:
Lets remove logs like these if they're not relevant after testing. Is this something that anyone needs to know on a regular basis?

Reply:
Removed.